diff --git a/dist/vue-select.es.js b/dist/vue-select.es.js
index 6a49eb6ac1e599868f530ba53760387b4eaffc1a..d780d3286f8fd103f7da637e79be23095a6d946b 100644
--- a/dist/vue-select.es.js
+++ b/dist/vue-select.es.js
@@ -632,7 +632,7 @@ https://vue-select.org/api/props.html#getoptionkey`, e, t);
         return n[e.keyCode](e);
     }
   }
-}, he = ["dir"], ce = ["id", "aria-expanded", "aria-owns"], ue = {
+}, he = ["dir"], ce = ["id"], ue = {
   ref: "selectedOptions",
   class: "vs__selected-options"
 }, pe = ["disabled", "title", "aria-label", "onClick"], fe = {
@@ -653,10 +653,6 @@ function ve(e, t, s, n, l, i) {
       id: `vs${s.uid}__combobox`,
       ref: "toggle",
       class: "vs__dropdown-toggle",
-      role: "combobox",
-      "aria-expanded": i.dropdownOpen.toString(),
-      "aria-owns": `vs${s.uid}__listbox`,
-      "aria-label": "Search for option",
       onMousedown: t[1] || (t[1] = (o) => i.toggleDropdown(o))
     }, [
       u("div", ue, [
